.github/workflows/main.yml

```diff
@@ -2391,7 +2391,7 @@ jobs:
     timeout-minutes: 60
     strategy:
       matrix:
-        java: [ '8' ]
+        java: [ '11' ]
```

Comment:
@dbalek @sdedic this would move the NBVSCode ext and the LSP job to JDK 11 since those jobs load all other modules. If the VSCode Ext has different requirements we would have to change the test setup there.

But from past conversations I believe to remember that JDK 11 was no problem there, right?